repos/system_upgrade/el7toel8/libraries/vsftpdutils.py

```diff
@@ -0,0 +1,32 @@
+import errno
+import hashlib
+
+from leapp.libraries.stdlib import api
+
+VSFTPD_CONFIG_DIR = '/etc/vsftpd'
+VSFTPD_DEFAULT_CONFIG_PATH = '/etc/vsftpd/vsftpd.conf'
+STRICT_SSL_READ_EOF = 'strict_ssl_read_eof'
+TCP_WRAPPERS = 'tcp_wrappers'
+
+
+def read_file(path):
+    with open(path, 'r') as f:
+        return f.read()
+
+
+def get_config_contents(path, read_func=read_file):
```

Comment:
```suggestion
def _get_config_contents(path, read_func=read_file):
```

Reply:
This turned out to be a public function :). I'll document it. I've renamed the other private functions though.

Reply:
I've documented it.